### ISSUE TITLE
Brew installation fails

### ISSUE THREAD

**Issue by aengl** [todotxt/todo.txt-cli#251]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Brew installation fails

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

```
$ brew install todo-txt
==> Downloading https://github.com/todotxt/todo.txt-cli/releases/download/v2.10.0/todo.txt_cli-2.10.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "todo-txt"
Download failed: https://github.com/todotxt/todo.txt-cli/releases/download/v2.10.0/todo.txt_cli-2.10.tar.gz
```

**Which Operating System are you using?**

High Sierra

**Which version of bash are you using?**
> Run `bash --version`

```
$ fish --version
fish, version 2.7.1
```

**Comment by karbassi** [comment]
Should be fixed once Homebrew/homebrew-core/pull/26307 is merged.

**Comment by aengl** [comment]
Works as of today.
